reactjs-eslint-prettier-warning-only-instead-of-error

Currently, webpack errors out when a linting error is made.
```
 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed


/home/ash/git/royal-hackaway/src/components/HackathonDefinition/index.js
  5:12  error  Replace `(⏎········<h1>Test</h1>⏎····)` with `<h1>Test</h1>`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.


File: src/components/HackathonDefinition/index.js

failed Re-building development bundle - 0.108s
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -33,6 +33,10 @@ module.exports = {
     // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/explicit-function-return-type.md
     // This rule means that TypeScript functions must always declare a type, even when it's obvious, such as `() => 'hello'`, to avoid incorrect type returns.
     "@typescript-eslint/explicit-function-return-type": "off",
+
+    // https://github.com/prettier/eslint-plugin-prettier
+    // This disables the errors for prettier, and replaces them with warnings. `-Wall`
+    "prettier/prettier": "warn",
   },
   overrides: [
     {

